### DISCUSSION TITLE
Asset prefix is invalid.

### DISCUSSION THREAD

**Discussion by goc-chushixian** [vercel/next.js#39846] [Help]
The asset prefix was valid when I started the project using 'yarn start', but when I went into the output standalone and started the project using 'node server.js', I found that the asset prefix was invalid. Why？
The following are the startup results using node server.js and yarn start
node server.js:
![KK6auzeAPQ](https://user-images.githubusercontent.com/107662661/186075845-ab363f48-a52e-4401-a766-23888001e1e2.jpg)
yarn start:
![AV0t4IsjY0](https://user-images.githubusercontent.com/107662661/186075996-9fc4d52a-a010-4ac4-823f-94c61df1f244.jpg)

**Comment by goc-chushixian** [comment]
I found the answer, I have a problem with the build command,
```
cross-env NODE_ENV=development && next build",
cross-env NODE_ENV=development  next build",
```
Using && causes the environment variable to change to production

**Reply by icyJoseph** [reply]
Why are you building as dev mode? Yeah, if you run with && is two different commands, whereas the second alternative sets the variable and runs the command next build in one process, so to speak.
